exercises/G-conditionals-2/exercise-2.js

```diff
@@ -7,7 +7,13 @@
 
 */
 
-function studentPassed(grade) {}
+function studentPassed(grade) {
+  if (grade < 50) {
+    return "failed";
+  } else if (grade >= 50) {
```

Comment:
You don't need a second if condition here. If you remove the (grade >= 50) this function will still work because you're not checking for multiple conditions.  You're only checking a single condition to see if the grade is under 50 or not.

Reply:
> You don't need a second if condition here. If you remove the (grade >= 50) this function will still work because you're not checking for multiple conditions. You're only checking a single condition to see if the grade is under 50 or not.

I have corrected this. Thank you